src/naive/date.rs

```diff
@@ -995,7 +995,8 @@ impl NaiveDate {
 
     /// Returns the packed month-day-flags.
     #[inline]
-    fn mdf(&self) -> Mdf {
+    #[doc(hidden)]
+    pub fn mdf(&self) -> Mdf {
```

Comment:
What problem does this solve? What API do you need from the `Mdf`, and can we support that from `NaiveDate` directly?

Reply:
This is to allow getting the day, month and year directly, but if we run the day, month and year then internally chrono have to get it from mdf multiple times, maybe the compiler can optimize it?

Reply:
This seems to be a micro-optimization then? Until we have a benchmark showing that this has a substantial performance impact, let's skip it.